ops/terraform/modules/database/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "db_username" {
+  type        = string
+  description = "Username of RDS Instance."
+  default     = "reportVisionDbUser"
+}
+
+variable "engine_version" {
+  description = "Postgres DB engine version."
+  default     = "11"
+}
+
+variable "location" {
+  type        = string
+  description = "Location of the resource."
+  default     = "eastus2"
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "The Azure Resource Group to deploy to"
+}
+
+# Designed for medium to high-performance workloads and is scalable.
+# May downsize to Standard_B1ms for development environments and small workloads.
+variable "sku_name" {
+  type        = string
+  description = "value"
+  default     = "GP_Standard_D2ds_v4" # General Purpose tier
```

Comment:
Even for dev environment we could 100% drop down to the smaller option. Our workloads are very lightweight.